### DISCUSSION TITLE
Cant make a backup

### DISCUSSION THREAD

**Discussion by Sabering1** [nextcloud/all-in-one#4911] [Questions] [answered]
### Steps to reproduce
1. Install SSD in debian linux latest, and mount it /mnt/backup
2. Click the backup button
3. gets error

### Expected behavior 
Backups data

### Actual behavior
Backup wont start

### Host OS
Debian stable latest

#### Nextcloud AIO version 
Latest

#### Current channel 
Stable

#### Other valuable info 
Clicking to see more info and i get this. it wont say what is wrong, it would be useful if it did
Performing backup...
Did not find borg.config file in the mastercontainer volume.
Cannot create a backup as this is wrong.

**Comment by szaimen** [comment, marked as answer]
Hi, maybe this helps? https://github.com/nextcloud/all-in-one/discussions/596
